source/Irrlicht/COpenGLDriver.h

```diff
@@ -874,7 +874,7 @@ namespace video
             //! Textures and Samplers
             class STextureStageCache : public core::AllocationOverrideDefault

```

Comment:
now that we have descriptor sets, this thing can be replaced in its entirety

(remember about the layout compatibility rules from vulkan spec!)

Reply:
http://vulkan-spec-chunked.ahcox.com/ch13s02.html

Reply:
Also, there needs to be two separate caches in `CNullDriver` one for compute and one for graphics pipeline (changing descriptors on one shouldn't influence the other), but COpenGLDriver must make it play with the fact that there are no separate bind points for graphics and compute (changing the set changes both)